src/main/duke/Duke.java

```diff
@@ -0,0 +1,115 @@
+package main.duke;
+import java.util.Scanner;
+
+
+public class Duke {
+    public static void main(String[] args) {
+        //Generates a greeting
+        greeting();
+        request();
+    }
+
+    public static void request(){
+        Scanner in = new Scanner(System.in);
+        Task[] list = new Task[100];
+        String line;
+        int i=0,j;
+        int task_count = 0;
+        while(true) {
+            String status ="[✗] ";
+            line = in.nextLine();
+
+            if (line.equals("bye")){
+                System.out.println("____________________________________________________________\n"
+                        +"Bye. Hope to see you again soon!\n"
+                +"____________________________________________________________\n");
+                break;
+            }else if(line.equals("list")){
+                for(j=0;j<i;j++){
+                    if(list[j].isDone){
+                        System.out.println((j+1)+ ". " + "[✓]" + list[j].description);
+                    }//not done
+                    else {
+                        System.out.println((j + 1) + ". " + "[✗] " + list[j].description);
+                    }
+                }
+            }else if(line.contains("done")){
+                int num = Integer.parseInt(line.substring(5));
+                System.out.println(num);
+                list[num-1].isDone = true;
+                System.out.println("____________________________________________________________\n"+
+                        "Nice! I've marked this task as done:");
+                System.out.println("[✓] "+ list[num-1].description + "\n____________________________________________________________\n");
```

Comment:
Perhaps it would be better if the long line of underscores can be replaced by a constant? This would also prevent the code from exceeding 120 characters per line, leading to coding standard violation.